src/main/java/org/broadinstitute/consent/http/db/UserDAO.java

```diff
@@ -122,12 +122,14 @@ WHERE LOWER(u.email) ILIKE ANY (array[<emails>])
       """)
   List<User> findUsersByEmailList(@BindList("emails") List<String> emails);
 
-  @SqlUpdate("INSERT INTO users (email, display_name, create_date) values (:email, :displayName, :createDate)")
+  @SqlUpdate("INSERT INTO users (email, display_name, institution_id, create_date) values (:email, :displayName, :institutionId, :createDate)")
   @GetGeneratedKeys
   Integer insertUser(@Bind("email") String email,
       @Bind("displayName") String displayName,
+      @Bind("institutionId") Integer institutionId,
       @Bind("createDate") Date createDate);
 
+  // Only used by tests; all users except Admin have an institution ID set on creation.
```

Comment:
I could remove API code completely changing tests to create two users: one to create the institute and another that uses the institute. Right now tests use one user for both purposes.